datafusion/src/physical_plan/datetime_expressions.rs

```diff
@@ -236,14 +236,8 @@ pub fn date_trunc(args: &[ColumnarValue]) -> Result<ColumnarValue> {
     let f = |x: Option<i64>| x.map(|x| date_trunc_single(granularity, x)).transpose();
 
     Ok(match array {
-        ColumnarValue::Scalar(scalar) => {
```

Comment:
is this change related to the avro addition?

Reply:
@nevi-me This is another clippy issue in nightly